devicetypes/plaidsystems/spruce-sensor.src/spruce-sensor.groovy

```diff
@@ -73,7 +73,7 @@ metadata {
 					[value: 64, color: "#44B621"],
 					[value: 80, color: "#3D79D9"],
 					[value: 96, color: "#0A50C2"]
-				]
+				], icon: 'http://www.plaidsystems.com/smartthings/st_spruce_leaf_225_top.png'
```

Comment:
can we use an existing humidity icon? if they change the link, this won't look pretty on the DTH